chart/testdata/frobnitz/LICENSE

```diff
@@ -0,0 +1,202 @@
+
```

Comment:
If we don't intend to release it independently (i.e., if it's part of this code base), then we don't need a separate license for it.